ts/client/features/transfer.ts

```diff
@@ -465,9 +471,39 @@ export class TransferPool implements ITransferPool {
 
 export class SimulatorPool extends OffchainTransferFactory
     implements ITransferPool {
-    private tokenID?: number;
+    private tokenID: number;
+    private initialBalance: ERC20Value;
     private feeReceiverID?: number;
-    async setTokenID() {
+
+    private readonly pubkey: PubkeyStorageEngine;
+
+    constructor(group: Group, storageManager: StorageManager) {
+        super(group, storageManager.state);
+        this.pubkey = storageManager.pubkey;
+
+        // TODO Make these configurable?
+        this.tokenID = 1;
+        this.initialBalance = USDT.fromHumanValue("100.12");
```

Comment:
@ChihChengLiang Will these every need to be configured from these defaults for future test cases? If so I can add an options object to the pool constructor.